packages/lambda-tiler/src/routes/tile.style.json.ts

```diff
@@ -175,12 +177,32 @@ export function tileSetToStyle(
     (Env.get(Env.PublicUrlBase) ?? '') +
     `/v1/tiles/${tileSet.name}/${tileMatrix.identifier}/{z}/{x}/{y}.${tileFormat}${query}`;
 
+  // attempt to load the tileset's imagery
+  const imagery = await (function (): Promise<ConfigImagery | null> {
```

Comment:
there may already a function that does this (inside the config, something like `getAllImagery()` )

You should also use `tileMatrix` over hard coding NZTM2000.